eternalcore-core/src/main/java/com/eternalcode/core/feature/afk/AfkService.java

```diff
@@ -27,8 +27,15 @@ public AfkService(AfkSettings afkSettings, NoticeService noticeService, UserMana
     public Afk markAfk(UUID player, AfkReason reason) {
         Afk afk = new Afk(player, reason, Instant.now());
 
-        this.afkByPlayer.put(player, afk);
-        this.notifyAfk(player, false);
+        if (!this.isAfk(player)) {
+            this.afkByPlayer.put(player, afk);
+            this.notifyAfk(player, true);
+
+            return afk;
+        }
+
+        this.clearAfk(player);
```

Comment:
Nie oczekiwana zmiana state, nie ma to sensu.

Reply:
Liczę że zaproponujesz lepsze rozwiązanie 🙃

Reply:
> Liczę że zaproponujesz lepsze rozwiązanie 🙃

Czekam na feedback

Reply:
> Się doczekasz 🙃

Mam nadzieje 💯